Ports/libsamplerate/patches/0001-Add-serenity-support-to-config.sub.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Mike Swanson <mikeonthecomputer@gmail.com>
+Date: Wed, 10 May 2023 18:46:42 -0700
+Subject: [PATCH] Add serenity support to config.sub
+
+---
+ build-aux/config.sub | 1 +
+ 1 file changed, 1 insertion(+)
+
+diff --git a/build-aux/config.sub b/build-aux/config.sub
+index 4d89efe3b69d3e7d2f34a06c2a252661fc29b8dd..b7e20b5093a586e748ce0b13211fcb132a306eed 100755
+--- a/build-aux/config.sub
++++ b/build-aux/config.sub
+@@ -1714,6 +1714,7 @@ case $os in
+ 	# The portable systems comes first.
+ 	# Each alternative MUST end in a * to match a version number.
+ 	gnu* | android* | bsd* | mach* | minix* | genix* | ultrix* | irix* \
++	     | serenity* \
+ 	     | *vms* | esix* | aix* | cnk* | sunos | sunos[34]* \
+ 	     | hpux* | unos* | osf* | luna* | dgux* | auroraux* | solaris* \
+ 	     | sym* |  plan9* | psp* | sim* | xray* | os68k* | v88r* \
```

Comment:
Same here: have you tried `use_fresh_config_sub`?

Reply:
You haven't changed this or responded to this discussion, so unresolving this :^)